src/main/java/org/symphonyoss/symphony/messageml/elements/TextNode.java

```diff
@@ -41,7 +41,7 @@ public TextNode(Element parent, String text) {
   @Override
   public void asPresentationML(XmlPrintStream out) {
     String content = this.text;
-    if (out.isRemoveNl() && !(getParent() instanceof Code)) { // Do not remove newlines if the parent node is a code node
+    if (out.isRemoveNl() && !(getParent() instanceof Code) && !(getParent() instanceof TextArea)) { // Do not remove newlines if the parent node is a code node
```

Comment:
Update this comment to contemplate TextArea as well

Reply:
done